WooCommerce/Classes/Extensions/WordPressSupportSourceTag+HelpCenter.swift

```diff
@@ -0,0 +1,18 @@
+import WordPressAuthenticator
+
+extension WordPressSupportSourceTag {
+    /// Returns the custom help content web page's URL of the screen with this (`self`) `sourceTag`.
+    ///
+    /// Returns `nil` if the screen with the current `sourceTag` doesn't have custom help content available.
+    ///
+    /// The `sourceTag` is set for login related screens in WordPressAuthenticator library.
+    ///
+    var customHelpCenterURL: URL? {
```

Comment:
nit: if we pass `sourceTag` to `HelpAndSupportViewController`, then this extension could probably be private inside the view controller since this knowledge isn't applicable to other use cases.